scene/gui/color_mode.cpp

```diff
@@ -262,6 +273,17 @@ bool ColorModeRAW::apply_theme() const {
 	return true;
 }
 
+void ColorModeOKHSL::_value_changed() {
+	Vector<float> values = color_picker->get_active_slider_values();
+
+	if (values[1] > 0 || values[0] != cached_hue) { // Cache hue
```

Comment:
```suggestion
	if (values[1] > 0 || values[0] != cached_hue) {
```
The comments are not needed after variables were renamed.